lib/dest/index.js

```diff
@@ -14,8 +15,9 @@ function dest(outFolder, opt) {
 
   var saveStream = pumpify.obj(
     prepare.dest(outFolder, opt),
+    makeDirs(opt),
```

Comment:
Oh, good catch. Perhaps a test for that would be worthwhile. I figured this order was equivalent except it would fail fast when mkdir goes awry.